codecov.yml

```diff
@@ -1,2 +1,6 @@
-ignore:
-  - "*/tests/*"
```

Comment:
Do we want to keep ignoring tests or it does not matter?

Reply:
I don't think it matters. I copied and pasted this config option when I set up codecov, but it doesn't actually seem to be affecting our coverage.